scripts/sanitycheck

```diff
@@ -3173,6 +3176,13 @@ def main():
     global run_individual_tests
     options = parse_arguments()
 
+    if (os.name == 'nt'):
+        print("Running sanitycheck on Windows is not supported yet.")
+        print("https://github.com/zephyrproject-rtos/zephyr/issues/2664")
+        if (not options.build_only):
+            print("Running sanitycheck on Windows with '--build-only' might work.")
```

Comment:
Why only "might work" ?

If this is useful enough to be upstreamed then we would want to do so to avoid conflicts.

Reply:
I tested windows sanitycheck on zephyr samples and it requires one more flag: '--disable-size-report'.   

I've added a check for this in the script.